tap_google_sheets/test.py

```diff
@@ -0,0 +1,31 @@
+import dateutil.parser
```

Comment:
What is the purpose of this file? I don't think it should be included.

Reply:
To use the `dateutil.parser.parse()` on [L10](https://github.com/singer-io/tap-google-sheets/pull/79/files#diff-0594dc805436cc5bf1a25bfa3d3e597e3daf988d3ccde3c497c6ad4953a56b97R10) in the function `strptime_to_utc`, we have imported the dateutil.parser.

Reply:
Why is the `test.py` file included in this PR?

Reply:
My bad, it was done by mistake. Removed the file.